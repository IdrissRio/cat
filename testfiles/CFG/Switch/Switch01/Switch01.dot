digraph G {
graph [splines=ortho, nodesep="1", ranksep="1"]
node [shape=rect, fontname=Arial];
{ rank = same; "Program[0]"}
{ rank = same; "List[1]"}
{ rank = same; "CompilationUnit[2]"}
{ rank = same; "List[3]"; "List[4]"}
{ rank = same; "ClassDecl[5]"}
{ rank = same; "Modifiers[22]"; "Opt[25]"; "List[26]"; "List[27]"; "InstanceInitialization[6]"; "InstanceInitialization[14]"}
{ rank = same; "List[23]"; "MethodDecl[28]"; "Entry[7]"; "Exit[8]"; "Entry[15]"; "Exit[16]"}
{ rank = same; "Modifier[24]"; "Entry[29]"; "Modifiers[30]"; "PrimitiveTypeAccess[32]"; "List[33]"; "List[34]"; "Opt[35]"; "Exit[36]"}
{ rank = same; "List[31]"; "Block[36]"}
{ rank = same; "List[37]"}
{ rank = same; "VarDeclStmt[38]"; "SwitchStmt[47]"}
{ rank = same; "Modifiers[39]"; "PrimitiveTypeAccess[41]"; "List[42]"; "AssignSimpleExpr[48]"; "Block[51]"}
{ rank = same; "List[40]"; "VariableDeclarator[43]"; "VarAccess[49]"; "IntegerLiteral[50]"; "List[52]"}
{ rank = same; "List[44]"; "Opt[45]"; "ConstCase[53]"; "Block[55]"; "ConstCase[61]"; "ExprStmt[65]"; "ExprStmt[69]"; "DefaultCase[73]"; "ExprStmt[74]"}
{ rank = same; "IntegerLiteral[46]"; "IntegerLiteral[54]"; "List[56]"; "AddExpr[62]"; "AssignSimpleExpr[66]"; "AssignSimpleExpr[70]"; "AssignSimpleExpr[75]"}
{ rank = same; "ExprStmt[57]"; "IntegerLiteral[63]"; "IntegerLiteral[64]"; "VarAccess[67]"; "IntegerLiteral[68]"; "VarAccess[71]"; "IntegerLiteral[72]"; "VarAccess[76]"; "IntegerLiteral[77]"}
{ rank = same; "VarAccess[59]"; "IntegerLiteral[60]"}
{ rank = same; "AssignSimpleExpr[58]"}
"Program[0]" [label="Program\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[1]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"CompilationUnit[2]" [label="CompilationUnit\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[3]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[4]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ClassDecl[5]" [label="ClassDecl\n < Switch01 >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[22]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[23]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifier[24]" [label="Modifier\n < public >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[25]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[26]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[27]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"MethodDecl[28]" [label="MethodDecl\n < m >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Entry[29]" [label="Entry\n ", fillcolor=white   style=filled  ]
"Modifiers[30]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[31]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[32]" [label="PrimitiveTypeAccess\n < void >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[33]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[34]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[35]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[36]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[37]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VarDeclStmt[38]" [label="VarDeclStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Modifiers[39]" [label="Modifiers\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[40]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"PrimitiveTypeAccess[41]" [label="PrimitiveTypeAccess\n < int >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[42]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"VariableDeclarator[43]" [label="VariableDeclarator\n < a >", fillcolor=white   style=filled  ]
"List[44]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Opt[45]" [label="Opt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"IntegerLiteral[46]" [label="IntegerLiteral\n < 0 >", fillcolor=white   style=filled  ]
"SwitchStmt[47]" [label="SwitchStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"AssignSimpleExpr[48]" [label="AssignSimpleExpr\n < a >", fillcolor=white   style=filled  ]
"VarAccess[49]" [label="VarAccess\n < a >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"IntegerLiteral[50]" [label="IntegerLiteral\n < 1 >", fillcolor=white   style=filled  ]
"Block[51]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[52]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ConstCase[53]" [label="ConstCase\n < 1 >", fillcolor=white   style=filled  ]
"IntegerLiteral[54]" [label="IntegerLiteral\n < 1 >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Block[55]" [label="Block\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"List[56]" [label="List\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"ExprStmt[57]" [label="ExprStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"AssignSimpleExpr[58]" [label="AssignSimpleExpr\n < a >", fillcolor=white   style=filled  ]
"VarAccess[59]" [label="VarAccess\n < a >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"IntegerLiteral[60]" [label="IntegerLiteral\n < 2 >", fillcolor=white   style=filled  ]
"ConstCase[61]" [label="ConstCase\n < org.extendj.ast.AddExpr >", fillcolor=white   style=filled  ]
"AddExpr[62]" [label="AddExpr\n < + >", fillcolor=white   style=filled  ]
"IntegerLiteral[63]" [label="IntegerLiteral\n < 1 >", fillcolor=white   style=filled  ]
"IntegerLiteral[64]" [label="IntegerLiteral\n < 1 >", fillcolor=white   style=filled  ]
"ExprStmt[65]" [label="ExprStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"AssignSimpleExpr[66]" [label="AssignSimpleExpr\n < a >", fillcolor=white   style=filled  ]
"VarAccess[67]" [label="VarAccess\n < a >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"IntegerLiteral[68]" [label="IntegerLiteral\n < 3 >", fillcolor=white   style=filled  ]
"ExprStmt[69]" [label="ExprStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"AssignSimpleExpr[70]" [label="AssignSimpleExpr\n < a >", fillcolor=white   style=filled  ]
"VarAccess[71]" [label="VarAccess\n < a >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"IntegerLiteral[72]" [label="IntegerLiteral\n < 4 >", fillcolor=white   style=filled  ]
"DefaultCase[73]" [label="DefaultCase\n ", fillcolor=white   style=filled  ]
"ExprStmt[74]" [label="ExprStmt\n ", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"AssignSimpleExpr[75]" [label="AssignSimpleExpr\n < a >", fillcolor=white   style=filled  ]
"VarAccess[76]" [label="VarAccess\n < a >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"IntegerLiteral[77]" [label="IntegerLiteral\n < 5 >", fillcolor=white   style=filled  ]
"Exit[36]" [label="Exit\n ", fillcolor=white   style=filled  ]
"InstanceInitialization[6]" [label="InstanceInitialization\n < Instance >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Entry[7]" [label="Entry\n ", fillcolor=white   style=filled  ]
"Exit[8]" [label="Exit\n ", fillcolor=white   style=filled  ]
"InstanceInitialization[14]" [label="InstanceInitialization\n < Static >", style= dotted   fillcolor="#eeeeee" fontcolor="#aaaaaa"  ]
"Entry[15]" [label="Entry\n ", fillcolor=white   style=filled  ]
"Exit[16]" [label="Exit\n ", fillcolor=white   style=filled  ]
"CompilationUnit[2]" -> "List[3]" [style=dashed, color=gray]
"List[23]" -> "Modifier[24]" [style=dashed, color=gray]
"Modifiers[22]" -> "List[23]" [style=dashed, color=gray]
"ClassDecl[5]" -> "Modifiers[22]" [style=dashed, color=gray]
"ClassDecl[5]" -> "Opt[25]" [style=dashed, color=gray]
"ClassDecl[5]" -> "List[26]" [style=dashed, color=gray]
"MethodDecl[28]" -> "Entry[29]" [style=dashed, color=gray]
"Modifiers[30]" -> "List[31]" [style=dashed, color=gray]
"MethodDecl[28]" -> "Modifiers[30]" [style=dashed, color=gray]
"MethodDecl[28]" -> "PrimitiveTypeAccess[32]" [style=dashed, color=gray]
"MethodDecl[28]" -> "List[33]" [style=dashed, color=gray]
"MethodDecl[28]" -> "List[34]" [style=dashed, color=gray]
"Modifiers[39]" -> "List[40]" [style=dashed, color=gray]
"VarDeclStmt[38]" -> "Modifiers[39]" [style=dashed, color=gray]
"VarDeclStmt[38]" -> "PrimitiveTypeAccess[41]" [style=dashed, color=gray]
"VariableDeclarator[43]" -> "List[44]" [style=dashed, color=gray]
"Opt[45]" -> "IntegerLiteral[46]" [style=dashed, color=gray]
"VariableDeclarator[43]" -> "Opt[45]" [style=dashed, color=gray]
"List[42]" -> "VariableDeclarator[43]" [style=dashed, color=gray]
"VarDeclStmt[38]" -> "List[42]" [style=dashed, color=gray]
"List[37]" -> "VarDeclStmt[38]" [style=dashed, color=gray]
"AssignSimpleExpr[48]" -> "VarAccess[49]" [style=dashed, color=gray]
"AssignSimpleExpr[48]" -> "IntegerLiteral[50]" [style=dashed, color=gray]
"SwitchStmt[47]" -> "AssignSimpleExpr[48]" [style=dashed, color=gray]
"ConstCase[53]" -> "IntegerLiteral[54]" [style=dashed, color=gray]
"List[52]" -> "ConstCase[53]" [style=dashed, color=gray]
"AssignSimpleExpr[58]" -> "VarAccess[59]" [style=dashed, color=gray]
"AssignSimpleExpr[58]" -> "IntegerLiteral[60]" [style=dashed, color=gray]
"ExprStmt[57]" -> "AssignSimpleExpr[58]" [style=dashed, color=gray]
"List[56]" -> "ExprStmt[57]" [style=dashed, color=gray]
"Block[55]" -> "List[56]" [style=dashed, color=gray]
"List[52]" -> "Block[55]" [style=dashed, color=gray]
"AddExpr[62]" -> "IntegerLiteral[63]" [style=dashed, color=gray]
"AddExpr[62]" -> "IntegerLiteral[64]" [style=dashed, color=gray]
"ConstCase[61]" -> "AddExpr[62]" [style=dashed, color=gray]
"List[52]" -> "ConstCase[61]" [style=dashed, color=gray]
"AssignSimpleExpr[66]" -> "VarAccess[67]" [style=dashed, color=gray]
"AssignSimpleExpr[66]" -> "IntegerLiteral[68]" [style=dashed, color=gray]
"ExprStmt[65]" -> "AssignSimpleExpr[66]" [style=dashed, color=gray]
"List[52]" -> "ExprStmt[65]" [style=dashed, color=gray]
"AssignSimpleExpr[70]" -> "VarAccess[71]" [style=dashed, color=gray]
"AssignSimpleExpr[70]" -> "IntegerLiteral[72]" [style=dashed, color=gray]
"ExprStmt[69]" -> "AssignSimpleExpr[70]" [style=dashed, color=gray]
"List[52]" -> "ExprStmt[69]" [style=dashed, color=gray]
"List[52]" -> "DefaultCase[73]" [style=dashed, color=gray]
"AssignSimpleExpr[75]" -> "VarAccess[76]" [style=dashed, color=gray]
"AssignSimpleExpr[75]" -> "IntegerLiteral[77]" [style=dashed, color=gray]
"ExprStmt[74]" -> "AssignSimpleExpr[75]" [style=dashed, color=gray]
"List[52]" -> "ExprStmt[74]" [style=dashed, color=gray]
"Block[51]" -> "List[52]" [style=dashed, color=gray]
"SwitchStmt[47]" -> "Block[51]" [style=dashed, color=gray]
"List[37]" -> "SwitchStmt[47]" [style=dashed, color=gray]
"Block[36]" -> "List[37]" [style=dashed, color=gray]
"Opt[35]" -> "Block[36]" [style=dashed, color=gray]
"MethodDecl[28]" -> "Opt[35]" [style=dashed, color=gray]
"MethodDecl[28]" -> "Exit[36]" [style=dashed, color=gray]
"List[27]" -> "MethodDecl[28]" [style=dashed, color=gray]
"ClassDecl[5]" -> "List[27]" [style=dashed, color=gray]
"InstanceInitialization[6]" -> "Entry[7]" [style=dashed, color=gray]
"InstanceInitialization[6]" -> "Exit[8]" [style=dashed, color=gray]
"InstanceInitialization[14]" -> "Entry[15]" [style=dashed, color=gray]
"InstanceInitialization[14]" -> "Exit[16]" [style=dashed, color=gray]
"ClassDecl[5]" -> "InstanceInitialization[6]" [style=dashed, color=gray]
"ClassDecl[5]" -> "InstanceInitialization[14]" [style=dashed, color=gray]
"List[4]" -> "ClassDecl[5]" [style=dashed, color=gray]
"CompilationUnit[2]" -> "List[4]" [style=dashed, color=gray]
"List[1]" -> "CompilationUnit[2]" [style=dashed, color=gray]
"Program[0]" -> "List[1]" [style=dashed, color=gray]
"Exit[16]" -> "Entry[15]" [color=red, constraint=false]
"Entry[15]" -> "Exit[16]" [color=blue, constraint=false, xlabel=" "] 
"Exit[8]" -> "Entry[7]" [color=red, constraint=false]
"Entry[7]" -> "Exit[8]" [color=blue, constraint=false, xlabel=" "] 
"Exit[36]" -> "AssignSimpleExpr[75]" [color=red, constraint=false]
"AssignSimpleExpr[75]" -> "Exit[36]" [color=blue, constraint=false, xlabel=" "] 
"AssignSimpleExpr[75]" -> "IntegerLiteral[77]" [color=red, constraint=false]
"IntegerLiteral[77]" -> "AssignSimpleExpr[75]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[77]" -> "DefaultCase[73]" [color=red, constraint=false]
"DefaultCase[73]" -> "IntegerLiteral[77]" [color=blue, constraint=false, xlabel=" "] 
"DefaultCase[73]" -> "ConstCase[61]" [color=red, constraint=false]
"DefaultCase[73]" -> "AssignSimpleExpr[70]" [color=red, constraint=false]
"AssignSimpleExpr[70]" -> "DefaultCase[73]" [color=blue, constraint=false, xlabel=" "] 
"AssignSimpleExpr[70]" -> "IntegerLiteral[72]" [color=red, constraint=false]
"IntegerLiteral[72]" -> "AssignSimpleExpr[70]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[72]" -> "AssignSimpleExpr[66]" [color=red, constraint=false]
"AssignSimpleExpr[66]" -> "IntegerLiteral[72]" [color=blue, constraint=false, xlabel=" "] 
"AssignSimpleExpr[66]" -> "IntegerLiteral[68]" [color=red, constraint=false]
"IntegerLiteral[68]" -> "AssignSimpleExpr[66]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[68]" -> "ConstCase[61]" [color=red, constraint=false]
"ConstCase[61]" -> "IntegerLiteral[68]" [color=blue, constraint=false, xlabel="T "] 
"ConstCase[61]" -> "DefaultCase[73]" [color=blue, constraint=false, xlabel="F "] 
"ConstCase[61]" -> "AddExpr[62]" [color=red, constraint=false]
"AddExpr[62]" -> "ConstCase[61]" [color=blue, constraint=false, xlabel=" "] 
"AddExpr[62]" -> "IntegerLiteral[64]" [color=red, constraint=false]
"IntegerLiteral[64]" -> "AddExpr[62]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[64]" -> "IntegerLiteral[63]" [color=red, constraint=false]
"IntegerLiteral[63]" -> "IntegerLiteral[64]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[63]" -> "ConstCase[53]" [color=red, constraint=false]
"IntegerLiteral[63]" -> "AssignSimpleExpr[58]" [color=red, constraint=false]
"AssignSimpleExpr[58]" -> "IntegerLiteral[63]" [color=blue, constraint=false, xlabel=" "] 
"AssignSimpleExpr[58]" -> "IntegerLiteral[60]" [color=red, constraint=false]
"IntegerLiteral[60]" -> "AssignSimpleExpr[58]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[60]" -> "ConstCase[53]" [color=red, constraint=false]
"ConstCase[53]" -> "IntegerLiteral[60]" [color=blue, constraint=false, xlabel="T "] 
"ConstCase[53]" -> "IntegerLiteral[63]" [color=blue, constraint=false, xlabel="F "] 
"ConstCase[53]" -> "AssignSimpleExpr[48]" [color=red, constraint=false]
"AssignSimpleExpr[48]" -> "ConstCase[53]" [color=blue, constraint=false, xlabel=" "] 
"AssignSimpleExpr[48]" -> "IntegerLiteral[50]" [color=red, constraint=false]
"IntegerLiteral[50]" -> "AssignSimpleExpr[48]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[50]" -> "VariableDeclarator[43]" [color=red, constraint=false]
"VariableDeclarator[43]" -> "IntegerLiteral[50]" [color=blue, constraint=false, xlabel=" "] 
"VariableDeclarator[43]" -> "IntegerLiteral[46]" [color=red, constraint=false]
"IntegerLiteral[46]" -> "VariableDeclarator[43]" [color=blue, constraint=false, xlabel=" "] 
"IntegerLiteral[46]" -> "Entry[29]" [color=red, constraint=false]
"Entry[29]" -> "IntegerLiteral[46]" [color=blue, constraint=false, xlabel=" "] 
"List[3]" -> "List[4]" [style=invis]
"InstanceInitialization[14]" -> "InstanceInitialization[6]" -> "Modifiers[22]" -> "Opt[25]" -> "List[26]" -> "List[27]" [style=invis]
"Entry[29]" -> "Modifiers[30]" -> "PrimitiveTypeAccess[32]" -> "List[33]" -> "List[34]" -> "Opt[35]" -> "Exit[36]" [style=invis]
"VarDeclStmt[38]" -> "SwitchStmt[47]" [style=invis]
"Modifiers[39]" -> "PrimitiveTypeAccess[41]" -> "List[42]" [style=invis]
"List[44]" -> "Opt[45]" [style=invis]
"AssignSimpleExpr[48]" -> "Block[51]" [style=invis]
"VarAccess[49]" -> "IntegerLiteral[50]" [style=invis]
"ConstCase[53]" -> "Block[55]" -> "ConstCase[61]" -> "ExprStmt[65]" -> "ExprStmt[69]" -> "DefaultCase[73]" -> "ExprStmt[74]" [style=invis]
"VarAccess[59]" -> "IntegerLiteral[60]" [style=invis]
"IntegerLiteral[63]" -> "IntegerLiteral[64]" [style=invis]
"VarAccess[67]" -> "IntegerLiteral[68]" [style=invis]
"VarAccess[71]" -> "IntegerLiteral[72]" [style=invis]
"VarAccess[76]" -> "IntegerLiteral[77]" [style=invis]
"Entry[7]" -> "Exit[8]" [style=invis]
"Entry[15]" -> "Exit[16]" [style=invis]

subgraph cluster_legend {
{node [style=filled, fillcolor=1, colorscheme="pastel13"]
Nodes [label="Total Nr. Nodes
69", fillcolor=1]
Edges [label="Total Nr. Edges
114", fillcolor=1]
NodesCFG [label="CFG Nr. Nodes
22", fillcolor=3]
EdgesCFG [label="CFG Nr. Edges
23", fillcolor=3]
Nodes-> "Program[0]" -> NodesCFG  [style="invis"]
Edges -> "Program[0]" ->EdgesCFG [style="invis"]}
label = "Statistics of the framework: NEXTFramework"
style="solid"
ranksep=0.05
nodesep=0.01
labelloc = b
len=2
}}