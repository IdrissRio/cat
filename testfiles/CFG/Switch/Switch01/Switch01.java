public class Switch01 {
  void m() {
    int a = 0;
    switch (a = 1) {
    case 1: {
      a = 2;
    }
    case 1 + 1:
      a = 3;
      a = 4;
    default:
      a = 5;
    }
  }
}