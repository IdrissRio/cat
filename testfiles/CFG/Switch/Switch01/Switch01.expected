NodeName: Entry[15] Successors: [Exit[16]]
NodeName: Exit[16] Successors: []
NodeName: Entry[7] Successors: [Exit[8]]
NodeName: Exit[8] Successors: []
NodeName: Entry[29] Successors: [IntegerLiteral[46]]
NodeName: IntegerLiteral[46] Successors: [VariableDeclarator[43]]
NodeName: VariableDeclarator[43] Successors: [IntegerLiteral[50]]
NodeName: IntegerLiteral[50] Successors: [AssignSimpleExpr[48]]
NodeName: AssignSimpleExpr[48] Successors: [ConstCase[53]]
NodeName: ConstCase[53] Successors: [IntegerLiteral[60], IntegerLiteral[63]]
NodeName: IntegerLiteral[60] Successors: [AssignSimpleExpr[58]]
NodeName: AssignSimpleExpr[58] Successors: [IntegerLiteral[63]]
NodeName: IntegerLiteral[63] Successors: [IntegerLiteral[64]]
NodeName: IntegerLiteral[64] Successors: [AddExpr[62]]
NodeName: AddExpr[62] Successors: [ConstCase[61]]
NodeName: ConstCase[61] Successors: [IntegerLiteral[68], DefaultCase[73]]
NodeName: IntegerLiteral[68] Successors: [AssignSimpleExpr[66]]
NodeName: AssignSimpleExpr[66] Successors: [IntegerLiteral[72]]
NodeName: IntegerLiteral[72] Successors: [AssignSimpleExpr[70]]
NodeName: AssignSimpleExpr[70] Successors: [DefaultCase[73]]
NodeName: DefaultCase[73] Successors: [IntegerLiteral[77]]
NodeName: IntegerLiteral[77] Successors: [AssignSimpleExpr[75]]
NodeName: AssignSimpleExpr[75] Successors: [Exit[36]]
NodeName: Exit[36] Successors: []

NodeName: Exit[16] Predecessor: [Entry[15]]
NodeName: Entry[15] Predecessor: []
NodeName: Exit[8] Predecessor: [Entry[7]]
NodeName: Entry[7] Predecessor: []
NodeName: Exit[36] Predecessor: [AssignSimpleExpr[75]]
NodeName: AssignSimpleExpr[75] Predecessor: [IntegerLiteral[77]]
NodeName: IntegerLiteral[77] Predecessor: [DefaultCase[73]]
NodeName: DefaultCase[73] Predecessor: [ConstCase[61], AssignSimpleExpr[70]]
NodeName: ConstCase[61] Predecessor: [AddExpr[62]]
NodeName: AddExpr[62] Predecessor: [IntegerLiteral[64]]
NodeName: IntegerLiteral[64] Predecessor: [IntegerLiteral[63]]
NodeName: IntegerLiteral[63] Predecessor: [ConstCase[53], AssignSimpleExpr[58]]
NodeName: ConstCase[53] Predecessor: [AssignSimpleExpr[48]]
NodeName: AssignSimpleExpr[48] Predecessor: [IntegerLiteral[50]]
NodeName: IntegerLiteral[50] Predecessor: [VariableDeclarator[43]]
NodeName: VariableDeclarator[43] Predecessor: [IntegerLiteral[46]]
NodeName: IntegerLiteral[46] Predecessor: [Entry[29]]
NodeName: Entry[29] Predecessor: []
NodeName: AssignSimpleExpr[58] Predecessor: [IntegerLiteral[60]]
NodeName: IntegerLiteral[60] Predecessor: [ConstCase[53]]
NodeName: AssignSimpleExpr[70] Predecessor: [IntegerLiteral[72]]
NodeName: IntegerLiteral[72] Predecessor: [AssignSimpleExpr[66]]
NodeName: AssignSimpleExpr[66] Predecessor: [IntegerLiteral[68]]
NodeName: IntegerLiteral[68] Predecessor: [ConstCase[61]]
